scripts/extract_test_db.sh

```
@@ -0,0 +1,6 @@
+# Ensures that the user has locally the db dir in /tmp.
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)